src/bidmanager.js

```diff
@@ -109,6 +109,13 @@ exports.addBidResponse = function (adUnitCode, bid) {
       utils.logError(errorMessage('No adUnitCode was supplied to addBidResponse.'));
       return false;
     }
+
+    const bidRequest = getBidderRequest(bid.bidderCode, adUnitCode);
+    if (!bidRequest.start) {
```

Comment:
if you trying to check for no bid request, I think this condition should be just ` if (!bidRequest)` otherwise you'll get an undefined error.